frontend/src/types.ts

```diff
@@ -181,6 +181,7 @@ export enum OdhDocumentType {
 }
 
 export type OdhDocument = {
+  kind?: string;
```

Comment:
🤔 This is likely something we need a tech debt issue for -- OdhDcouments are K8s Kinds today. But I bet this is because we generate them... so not much we can do today about this.